source/slang/core.meta.slang

```diff
@@ -1279,6 +1279,12 @@ struct Ptr
         [nonmutating]
         ref;
     }
+
+    [ForceInline]
```

Comment:
Instead of doing this, we should just make pointer type a C-Style type so they can be default initialized to 0.

Reply:
The difference is that is ptr is a c-style types, then the following will also work:
```
struct M
{
       int* x;
};

M m = {}; // should init.
```
If we only have `__init` on `Ptr` type, at the moment `x` won't be initialized unless it is defined as `int* x = {}`.